Bio/Cluster/cluster.c

```diff
@@ -37,14 +37,29 @@
 /* ************************************************************************ */
 
 double
-mean(int n, double x[])
+mean(int n, double a[])
+// Add 4 elements at once instead of 1. The advantages are:
+// 1. less loop overhead
+// 2. compile with -O2 -> use SSE/AVX. 
+// 3. without AVX, still faster because 4 independent additions -> parallel instruction possible
+// 4. smaller floating-point error
 {
```

Comment:
For consistency, please use `/* ... */ ` for comments instead of `//`.

Reply:
I have switched to /* ... */